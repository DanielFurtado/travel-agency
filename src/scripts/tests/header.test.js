import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/header';

describe('Header', () => {
  const header = shallow(<Header />);

  it('render properly', () => {
    expect(header).toMatchSnapshot();
  });
});