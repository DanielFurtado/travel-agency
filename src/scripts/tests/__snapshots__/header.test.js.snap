// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="header-wrapper"
>
        <img
                alt="logo"
                className="travel-agency-logo"
                src="logo_large.png"
        />
        <span
                className="header-title"
        >
                Travel Agency
        </span>
</div>,
      "className": "header",
      "id": "header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="logo"
            className="travel-agency-logo"
            src="logo_large.png"
/>,
          <span
            className="header-title"
>
            Travel Agency
</span>,
        ],
        "className": "header-wrapper",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "logo",
            "className": "travel-agency-logo",
            "src": "logo_large.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Travel Agency",
            "className": "header-title",
          },
          "ref": null,
          "rendered": "Travel Agency",
          "type": "span",
        },
      ],
      "type": "div",
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="header-wrapper"
>
          <img
                    alt="logo"
                    className="travel-agency-logo"
                    src="logo_large.png"
          />
          <span
                    className="header-title"
          >
                    Travel Agency
          </span>
</div>,
        "className": "header",
        "id": "header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="logo"
              className="travel-agency-logo"
              src="logo_large.png"
/>,
            <span
              className="header-title"
>
              Travel Agency
</span>,
          ],
          "className": "header-wrapper",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "logo",
              "className": "travel-agency-logo",
              "src": "logo_large.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Travel Agency",
              "className": "header-title",
            },
            "ref": null,
            "rendered": "Travel Agency",
            "type": "span",
          },
        ],
        "type": "div",
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
